packages/qcheck-ounit/qcheck-ounit.0.11/opam

```diff
@@ -0,0 +1,30 @@
+opam-version: "2.0"
+synopsis: "OUnit backend for qcheck"
+maintainer: "simon.cruanes.2007@m4x.org"
+authors: "Simon Cruanes <simon.cruanes.2007@m4x.org>"
+tags: ["qcheck" "quickcheck" "ounit"]
+homepage: "https://github.com/c-cube/qcheck/"
+doc: "http://c-cube.github.io/qcheck/"
+bug-reports: "https://github.com/c-cube/qcheck/issues"
+depends: [
+  "dune"
+  "base-bytes"
+  "base-unix"
+  "qcheck-core" {>= "0.9"}
```

Comment:
```suggestion
  "qcheck-core" {= version}
```
Are you sure this dependency constraint is correct? It might also avoid some future incompatibility fixes